protocols/luci-proto-3g/htdocs/luci-static/resources/protocol/3g.js

```diff
@@ -142,7 +142,7 @@ return network.registerProtocol('3g', {
 
 		o = s.taboption('advanced', form.Value, '_keepalive_interval', _('LCP echo interval'), _('Send LCP echo requests at the given interval in seconds, only effective in conjunction with failure threshold'));
 		o.placeholder = '1';
-		o.datatype    = 'min(1)';
+		o.datatype    = 'and(uinteger,min(1))';
```

Comment:
This is redundant: since `min(1)` keeps it above zero (`uinteger`) anyway.

Reply:
unless you can show me an example where it's necessary? (it executes more code)

Reply:
It is actually not redundant since `min(1)` allows floating point as well. It has to be an integer.

Reply:
Ah, of course. Floats.